tests/infer/test_enum.py

```diff
@@ -49,6 +50,7 @@ def model():
 def test_iter_discrete_traces_vector(graph_type):
```

Comment:
what's the deal with test_iter_discrete_traces_vector?

Reply:
Fixed and removed the `@xfail`